### PR TITLE
Update CHANGELOG and bump version for release

### PR DESCRIPTION
**Pull request by mathieujobin**

### CODE REVIEW

File: lib/prawn/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Prawn
-  VERSION = '2.4.0'
+  VERSION = '2.5.0'
```
Comment (pointlessone) [inline lib/prawn/version.rb:4]: I understand the sentiment but generally this is not done in a PR. I have not decided yet on the version that's going to be released.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,12 +2,50 @@
...
+
 ### Look for glyph in correct font
 
 Take the font style into account when looking for a glyph and fallback fonts are enabled.
 
 (Dan Allen, [#1147](https://github.com/prawnpdf/prawn/issues/1147))
 
+### Raise an exception on invalid valign value
+
+(Masakinpo)
```
Comment (pointlessone) [inline CHANGELOG.md:43]: Don't we have PRs associated with these changes? I might've been cheeky and push to master once ina while but most of the contributions probably had a PR opened first.

Comment (mathieujobin) [reply CHANGELOG.md:43]: the way things was merged, this information is really hard to track. I'm not going to spend the time tracking down this past 3 years

File: lib/prawn/version.rb (hunk 1 above)
Comment (mathieujobin) [reply lib/prawn/version.rb:4]: Right makes sense. I put my reasoning there. https://github.com/prawnpdf/prawn/issues/1195#issuecomment-1797836020

**Pull request closed without merging**